sst.config.ts

```diff
@@ -0,0 +1,26 @@
+import { SSTConfig } from 'sst'
+import { Tags } from 'aws-cdk-lib'
+import { LinkdexStack } from './stacks/LinkdexStack'
+
+export default {
+  config (_input) {
+    return {
+      name: 'linkdex-api',
+      region: 'us-east-2'
```

Comment:
worth double checking that the mechanism to override this default by an env var hasn't changed in new sst. It'd be suprising, but we don't want to accidentally publish prod to the dev region. it's set in seed.run env config currently